src/closure.rs

```diff
@@ -93,7 +93,7 @@ impl Closure {
         let mut result = unsafe { Value::uninitialized() };
 
         let v_args: Vec<Value>;
-        let mut s_args: [Value; 10] = unsafe { mem::zeroed() };
+        let mut s_args: [Value; 10] = unsafe { mem::MaybeUninit::zeroed().assume_init() };
```

Comment:
Same. See the example about arrays in the `MaybeUninit` documentation. Or keep this as `mem::zeroed()` if that's not applicable here.

But `assume_init()` right after creating the `MaybeUninit` is an anti-pattern and wrong in 99% of the cases.

Reply:
Just keep this `mem::zeroed()` if you can't nicely use `MaybeUninit` here :)